docs/contributing/index.md

```diff
@@ -83,6 +83,11 @@ You can submit pull requests for small changes such as:
 - Fixing warnings detected by compilers or analysis tools
 - Making small changes to a single package
 
-If your pull request is a large change not covered by the examples above, discuss with the maintainers before submission. Doing so ensures that the proposed change is in line with Autoware's design philosophy and current development plans. If you're not sure where to have that conversation, then [create a new Q&A discussion](https://github.com/autowarefoundation/autoware/discussions/new?category=q-a).
+If your pull request is a large change such as
+
+- Adding a new feature to Autoware
+- Adding a new page to the documentation
+
+discuss with the maintainers before submission. Doing so ensures that the proposed change is in line with Autoware's design philosophy and current development plans. If you're not sure where to have that conversation, then [create a new Q&A discussion](https://github.com/autowarefoundation/autoware/discussions/new?category=q-a).
```

Comment:
I think that it is better to provide examples at the end (this also avoids fragmenting the first sentence). How about this instead?

```suggestion
If your pull request is a large change, discuss with the maintainers before submission. Doing so ensures that the proposed change is in line with Autoware's design philosophy and current development plans. If you're not sure where to have that conversation, then [create a new Q&A discussion](https://github.com/autowarefoundation/autoware/discussions/new?category=q-a).

Examples of large changes include:
- Adding a new feature to Autoware
- Adding a new documentation page or section
```

Reply:
Looks good!